modules/http/src/main/java/org/glassfish/grizzly/http/ChunkedTransferEncoding.java

```diff
@@ -49,6 +49,9 @@ public final class ChunkedTransferEncoding implements TransferEncoding {
     private static final byte[] LAST_CHUNK_CRLF_BYTES = "0\r\n".getBytes(ASCII_CHARSET);
     private static final int[] DEC = HexUtils.getDecBytes();
 
+    public static final String STRICT_CHUNKED_TRANSFER_CODING_LINE_TERMINATOR_RFC_9112 = "org.glassfish.grizzly.http.STRICT_CHUNKED_TRANSFER_CODING_LINE_TERMINATOR_RFC_9112";
+    private static final boolean isStrictChunkedTransferCodingLineTerminatorSet = Boolean.parseBoolean(System.getProperty(STRICT_CHUNKED_TRANSFER_CODING_LINE_TERMINATOR_RFC_9112));
```

Comment:
Would you please move this initialization from system property to the constructor and change the static field to instance field?

With that, it's more deterministic when the system property is read and the system property can be reliably set from code before creating the instance, not only using a JVM option.